modules/articles/client/controllers/admin/list-articles.client.controller.js

```diff
@@ -10,6 +10,63 @@
   function ArticlesAdminListController(ArticlesService) {
     var vm = this;
 
-    vm.articles = ArticlesService.query();
+    vm.error = null;
+    vm.find = find;
+    vm.filter = filter;
+    vm.reset = reset;
+
+    activate();
+
+    function activate() {
+      vm.itemsPerPage = 10;
+      vm.currentPage = 1;
+      vm.filters = [];
+      vm.sorting = '-created';
```

Comment:
The point is that you may have multiple views, or user inputs, that change the filters and/or sorts.

Furthermore, you could have many different clients sending API requests to the back-end with varying filters & sorts.